contracts/ENSGovernorV2.sol

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/governance/Governor.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorProposalThreshold.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
+
+import "./EnsCounting.sol";
+import "./ENSGovernor.sol";
+
+/**
+ * @title ENSGovernorV2
+ * @dev Enhanced ENS Governor with bonding functionality.
+ */
+contract ENSGovernorV2 is ENSGovernor {
+    uint256 public availableBondsBalance;
```

Comment:
is there a specific reason for this variable? also, the naming feels a bit confusing because this amount is not necessarily available.

Reply:
I renamed the variable; it keeps track of the amount of forfeit tokens